kyuubi-common/src/test/scala/org/apache/kyuubi/HudiSuiteMixin.scala

```diff
@@ -38,7 +38,6 @@ trait HudiSuiteMixin extends DataLakeSuiteMixin {
   }
 
   override protected def extraConfigs = Map(
-    "spark.sql.catalogImplementation" -> "in-memory",
```

Comment:
this config is necessary

Reply:
Yeah, I see, just move it to `WithSparkSQLEngine`. 
BTW, why hudi doesn't work with `hive` catalog? `in-memory` should only used for test, I think most users would use hive for production.

Reply:
It work well if hive metastore is base on mysql  and will cause Concurrent Exception  if base on derby. 
Derby is the default config value, and ut is base on it .

Reply:
How does Hudi community do such a Hive integration test?

Reply:
```
spark.sql("...")
```
It works well with derby.